app/models/miq_alert_status_action.rb

```diff
@@ -0,0 +1,30 @@
+class MiqAlertStatusAction < ApplicationRecord
```

Comment:
We've been overriding active record callbacks.
I'm not thrilled with this method, but that is what we've been doing.
Maybe @Fryguy or @chrisarcand has an alternative way